src/app/icd/ICDManager.cpp

```diff
@@ -40,6 +40,16 @@ using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::IcdManagement;
 
+uint8_t ICDManager::OpenExchangeContextCount = 0;
+static_assert(UINT8_MAX >= CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS,
+              "ICDManager::OpenExchangeContextCount cannot hold count for the max exchange count");
+
+ICDNotifier * GetICDNotifier()
```

Comment:
Should this be in a separate cpp file for ICDNotifier instead of ICDManager? that way maybe we can make ICDNotifier not be all methods inline and we can put them in the cpp file as well.